### ISSUE TITLE
JupyterLab 4 update

### ISSUE THREAD

**Issue by dolgalevgv** [dunovank/jupyterlab_legos_ui#5]
Hi, thanks for making my favorite Jupyter theme! 

Any chance you could spare some time to update to support the newly released JupyterLab 4 (https://jupyterlab.readthedocs.io/en/latest/getting_started/changelog.html#v4-0)?

**Comment by dunovank** [comment]
This isn't the legos theme, but I recently created a new version of darkside for JupyterLab>=4 https://github.com/dunovank/jupyterlab_darkside_theme

**Comment by Winnus** [comment]
Is it the case that the legos and darkside UI/Theme are generally the same but have just switched colors, or are there more differences?

**Comment by dunovank** [comment]
The [jupyterlab_darkside_theme](https://github.com/dunovank/jupyterlab_darkside_theme) version of darkside supports JupyterLab 4, whereas [jupyterlab_darkside_ui](https://github.com/dunovank/jupyterlab_darkside_ui) and [jupyterlab_legos_ui](https://github.com/dunovank/jupyterlab_legos_ui) only support up to JupyterLab 3.*. Aside from that major difference there are likely a few stylistic differences between legos and darkside but I try to keep them to a minimum.

**Comment by dunovank** [comment]
I haven't created a new Legos UI with JupyterLab 4 support yet due to it being a less popular theme but if there's enough interest I can put this on the old #ToDo list

**Comment by Winnus** [comment]
Yes sorry I was not clear on my comment, sure the ui's are for Jupyterlab < 4 , the themes for >= 4.

I really enjoy the white variant, actually that much that I resisted to update Juppyterlab.
Especially the font change is a huge difference for me and the overall look makes it a pleasure to work with.

Unfortunately I have no experience in Jupyter themes.
Since you wrote that there are not to much differences:
I start by taking the values from the` legos_ui/style/variables.css` and put them into the   `darkside_theme/style/variables.css` to adapt darkside into legos ?

**Comment by aicaffeinelife** [comment]
The jupyterlab legos theme is my favorite. It was not until Jupyterlab 4.0 that I was forced to change it back to original (which sucks). If at all possible please make a port of this theme for `>=4.0` 😄
